index.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const fs = require('fs');
+
+
+fs.readFile(`${__dirname}/data/one.txt`, function(err, data) {
+  if (err) throw err;
+  console.log('content1:', data.toString('hex', 0, 8));
+
+  fs.readFile(`${__dirname}/data/two.txt`, function(err, data) {
+    if (err) throw err;
+    console.log('content2:', data.toString('hex', 0, 8));
+
+    fs.readFile(`${__dirname}/data/three.txt`, function(err, data) {
+      if (err) throw err;
+      console.log('content3:', data.toString('hex', 0, 8));
+    });
+  });
+});
```

Comment:
You could have also abstracted this into its own helper file and wrapped it in a function. Then pass that function a callback, and include an empty array within the function, then replace the console.logs with <someArray>.push(data.toString('hex', 0, 8))

At the end of all of the nested madness you could then pass that array to the callback you passed to the containing function.

In this way, you would be able to simply invoke that function from the helper file in index.js, being mindful of the callback.